package com.example.travelVar3.repository;

import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;

import com.example.travelVar3.entity.User;

public interface UserRepository extends JpaRepository<User, Integer>{
public User findByEmail(String email);	
public Page<User>findByNameLikeOrFuriganaLike(String nameKeyword,String furiganaKeyword,Pageable pageable);
}
